src/sprout/Helpers/CustomHeadTags.php

```diff
@@ -0,0 +1,421 @@
+<?php
+/*
+ * Copyright (C) 2023 Karmabunny Pty Ltd.
+ *
+ * This file is a part of SproutCMS.
+ *
+ * SproutCMS is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU General Public License as published by the Free Software Foundation, either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * For more information, visit <http://getsproutcms.com>.
+ */
+
+namespace Sprout\Helpers;
+
+use Kohana;
+
+/**
+ * Management of custom HEAD tags. Often meta but can also be link, base, etc
+ */
+class CustomHeadTags
+{
+
+    /**
+     * A list of possible tag combinations in the format of tag_type => [tags] => [values]
+     *
+     * NOTE: This can be customised by overriding the same setup in Kohana::config('sprout.custom_meta.available_tag_list')
+     *
+     * @var array
+     */
+    protected static $available_tag_list = [
```

Comment:
I recall kohana configs are supposed to have this sort of 'default/override' behaviour built-in. But I suspect it works differently for the sprout config. Shame.

Reply:
Nah maybe you're right, we can add a core config for it, that a user config would override